scripts/index.js

```diff
@@ -22,4 +22,17 @@ function evenNumbers(arr) {
     return evenArray;
 }
 
-console.log(evenNumbers([1, 4, 5, 6, 7]));
\ No newline at end of file
+console.log(evenNumbers([1, 4, 5, 6, 7]));
+
+// Square the Numbers
+// Write a function which takes an array of numbers as input and returns a new array containing result of squaring each of the numbers in the given array by two. Example: squareTheNumbers([1, 2, 3]) should give [1, 4, 9].
+
+function squaredNumbers(arr) {
+    const squaredArray = arr.map(function (num) {
```

Comment:
Solid technique. Maybe consider `returning` the result of `arr.map` directly? (That is, omit the variable.)